lib/model/katex.dart

```diff
@@ -136,6 +133,123 @@ class _KatexParser {
   KatexNode _parseSpan(dom.Element element) {
     // TODO maybe check if the sequence of ancestors matter for spans.
 
+    if (element.className.startsWith('strut')) {
+      if (element.className == 'strut' && element.nodes.isEmpty) {
```

Comment:
Is the `startsWith` logic necessary? Seems like we could just skip to the equality check.